pkg/api/aim/experiments.go

```diff
@@ -192,18 +222,21 @@ func GetExperimentActivity(c *fiber.Ctx) error {
 		ID string `params:"id"`
 	}{}
 
-	if err := c.ParamsParser(&p); err != nil {
+	if err = c.ParamsParser(&p); err != nil {
 		return fiber.NewError(fiber.StatusUnprocessableEntity, err.Error())
 	}
 
 	id, err := strconv.ParseInt(p.ID, 10, 32)
 	if err != nil {
 		return fiber.NewError(fiber.StatusUnprocessableEntity, fmt.Sprintf("unable to parse experiment id %q: %s", p.ID, err))
 	}
-	id32 := int32(id)
 
-	if tx := database.DB.Select("ID").First(&database.Experiment{
-		ID: &id32,
+	if tx := database.DB.Select(
+		"ID",
+	).Where(
+		"experiments.namespace_id = ?", ns.ID,
```

Comment:
Move this into the `Experiment` struct below

Reply:
done.